js/productsearch.js

```diff
@@ -0,0 +1,30 @@
+const { sendGetRequest, sendPostRequest } = require('./common');
+
+const END_POINT = 'https://search.visenze.com/v1/';
```

Comment:
the CN endpoint is search.visenze.com.cn, need to have a method to select which one